container-src/pac-deployer/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM centos:7
```

Comment:
We could think about making this based on alpine at some point to make it a bit lighter.